index.php

```diff
@@ -0,0 +1,311 @@
+<?php
+    if(!$_GET["page"]){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Detected usage of a non-sanitized input variable: $_GET["page"]](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)